### PR TITLE
modify create, update methods in domain and project dashboard

### PR DESCRIPTION
**Pull request by stat-kwon**
### Category
- [ ] New feature
- [ ] Bug fix
- [ ] Improvement
- [x] Refactor
- [ ] etc

### Description
There is a lot of unnecessary code to understand the business logic, 
so it is processed as an inner function to improve readability.

### Known issue
* #12

### CODE REVIEW

File: src/spaceone/dashboard/service/domain_dashboard_service.py [hunk 1]
```diff
@@ -49,7 +49,8 @@ def create(self, params):
         domain_dashboard_vo = self.domain_dashboard_mgr.create_domain_dashboard(params)
 
         version_keys = ['layouts', 'dashboard_options', 'dashboard_options_schema']
-        if set(version_keys) <= params.keys():
+        if any(key for key in params
```
Comment (whdalsrnt) [inline src/spaceone/dashboard/service/domain_dashboard_service.py:52]: I suggest improving the code here as follows.
`any(set(version_keys) & set(params.keys()))`

File: src/spaceone/dashboard/service/domain_dashboard_service.py [hunk 2]
```diff
@@ -81,29 +82,16 @@ def update(self, params):
...
-            else:
-                params['settings'].update({'date_range': {'enabled': domain_dashboard_vo.settings.date_range.enabled}})
-
-            if 'currency' in params['settings']:
-                params['settings']['currency'] = {
-                    'enabled': params['settings']['currency'].get('enabled', False)
-                }
-            else:
-                params['settings'].update({'currency': {'enabled': domain_dashboard_vo.settings.currency.enabled}})
+            self._create_settings_in_params(domain_dashboard_vo, params['settings'])
```
Comment (whdalsrnt) [inline src/spaceone/dashboard/service/domain_dashboard_service.py:91]: Clearly declare the results of function execution.
params['settings'] = self._create_settings_in_params(domain_dashboard_vo, params['settings'])

File: src/spaceone/dashboard/service/project_dashboard_service.py [hunk 3]
```diff
@@ -50,7 +50,8 @@ def create(self, params):
         project_dashboard_vo = self.project_dashboard_mgr.create_project_dashboard(params)
 
         version_keys = ['layouts', 'dashboard_options', 'dashboard_options_schema']
-        if set(version_keys) <= params.keys():
+        if any(key for key in params
```
Comment (whdalsrnt) [inline src/spaceone/dashboard/service/project_dashboard_service.py:53]: I suggest improving the code here as follows.
any(set(version_keys) & set(params.keys()))

**Review by whdalsrnt — COMMENTED** [review]
I suggest improving the code style.

**Comment by stat-kwon** [comment]
The following two code review improvements have been applied.
* Use set to find commonalities between two dictionaries
* Specifying results clearly

@whdalsrnt

**Review by whdalsrnt — APPROVED** [review]
Thanks for the nice code improvements.

**Pull request merged**